### PR TITLE
use podutils for verify on k/k master branch

### PR DESCRIPTION
**Pull request by BenTheElder**
/assign @krzyzacy 

FYI @fejta, hopefully a smooth transition

### CODE REVIEW

**Comment by krzyzacy** [comment]
we can switch integration on master as well.

cc @cblecker @ixdy

**Comment by BenTheElder** [comment]
I need this one more :-)

File: config/jobs/kubernetes/sig-testing/verify.yaml [hunk 1]
```diff
@@ -26,21 +28,20 @@ presubmits:
...
           requests:
             cpu: 4
-
   # TODO(krzyzacy): add branches once this works
   # TODO(krzyzacy): Consider combine kubekins-e2e and kubekins-test
-  - name: pull-kubernetes-verify-podutil
+  - name: pull-kubernetes-verify
     always_run: false
     decorate: true
-    branch:
```
Comment (ixdy) [inline config/jobs/kubernetes/sig-testing/verify.yaml]: I love YAML sooo much

File: config/jobs/kubernetes-security/generated-security-jobs.yaml [hunk 2]
```diff
@@ -2522,23 +2524,25 @@ presubmits:
...
+    rerun_command: /test pull-security-kubernetes-verify
     spec:
       containers:
       - args:
         - --ssh=/etc/ssh-security/ssh-security
         - ./hack/jenkins/verify-dockerized.sh
         command:
         - runner.sh
-        image: gcr.io/k8s-testimages/kubekins-e2e:latest-master
+        image: gcr.io/k8s-testimages/kubekins-e2e:v20190110-c23317e88
```
Comment (ixdy) [inline config/jobs/kubernetes-security/generated-security-jobs.yaml]: shouldn't these have a `-master`?

Comment (krzyzacy) [reply config/jobs/kubernetes-security/generated-security-jobs.yaml]: gooooooood catch :joy:

File: config/jobs/kubernetes/sig-testing/verify.yaml (hunk 1 above)
Comment (BenTheElder) [reply config/jobs/kubernetes/sig-testing/verify.yaml]: we could switch to strict parsing, and we should

File: config/jobs/kubernetes-security/generated-security-jobs.yaml (hunk 2 above)
Comment (BenTheElder) [reply config/jobs/kubernetes-security/generated-security-jobs.yaml]: done, yeah I copied this blindly from the other job

File: config/jobs/kubernetes/sig-testing/verify.yaml [hunk 3]
```diff
@@ -5,9 +5,11 @@ presubmits:
     labels:
       preset-service-account: "true"
       preset-dind-enabled: "true"
+    skip_branches:
+    - master # the podutils version below does this branch now
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/bootstrap:v20190110-c23317e88
+      - image: gcr.io/k8s-testimages/bootstrap:v20190110-c23317e88-master
```
Comment (ixdy) [inline config/jobs/kubernetes/sig-testing/verify.yaml]: this one is bootstrap, so no `-master` needed?

Comment (krzyzacy) [inline config/jobs/kubernetes/sig-testing/verify.yaml]: bootstrap doesn't have -master :-)

File: config/jobs/kubernetes/sig-testing/verify.yaml [hunk 4]
```diff
@@ -26,21 +28,20 @@ presubmits:
         resources:
           requests:
             cpu: 4
-
   # TODO(krzyzacy): add branches once this works
   # TODO(krzyzacy): Consider combine kubekins-e2e and kubekins-test
-  - name: pull-kubernetes-verify-podutil
+  - name: pull-kubernetes-verify
     always_run: false
```
[the remark this answers is not in the document]
Comment (BenTheElder) [reply config/jobs/kubernetes/sig-testing/verify.yaml]: yeaaaah. context is offline I discovered this on one of my PRs and brought it up with @krzyzacy and @ixdy ... not good. filed https://github.com/kubernetes/test-infra/pull/10718 to fix

**Pull request merged**